src/frontend/utils/resumeLive.ts

```diff
@@ -0,0 +1,38 @@
+// module responsible to fetch an HLS manifest and detect when tag
+// EXT-X-ENDLIST has been removed in it.
+import { Parser } from 'm3u8-parser';
+
+import { getResource } from 'data/sideEffects/getResource';
+import { modelName } from 'types/models';
+import { Video } from 'types/tracks';
+
+const regex = /^(https.*)\/.*\.m3u8$/;
+
+export const resumeLive = async (video: Video) => {
+  const manifestUrl = video.urls?.manifests.hls!;
```

Comment:
This is more a long term thought, but do we want to use assertion or should we create an issue to type AppData and DecodedJwt in a way we can remove type assertions ?

(this is also true with cast)

Reply:
It is a good idea to open an issue to type AppData and DecodedJwt :+1: